assets/dev/scss/frontend/_container.scss

```diff
@@ -34,6 +34,9 @@
 	--overlay-mix-blend-mode: initial;
 	--overlay-opacity: 1;
 	--overlay-transition: 0.3s;
+	--grid-justify-content: start;
```

Comment:
@ArmanPurtoian please nest these variables under e-grid class. The flex container does not need these variables.